packages/run-protocol/src/vpool-xyk-amm/multipoolMarketMaker.js

```diff
@@ -161,8 +165,18 @@ const start = async (zcf, privateArgs) => {
   const quoteIssuerKit = makeIssuerKit('Quote', AssetKind.SET);
 
   /** @type {SubscriptionRecord<MetricsNotification>} */
-  const { publication: metricsPublication, subscription: metricsSubscription } =
-    makeSubscriptionKit();
+  const {
+    publication: metricsPublication,
+    subscription: rawMetricsSubscription,
+  } = makeSubscriptionKit();
+  const { storageNode, marshaller } = privateArgs;
+  const metricsStorageNode =
+    storageNode && E(storageNode).getChildNode('metrics'); // TODO: magic string
```

Comment:
maybe change the "TODO: magic string" comment to a justification: 'metrics' is derived from `getMetrics` which is already part of the public API. I suppose it's still feasible to make a manifest constant.

Reply:
I left this TODO intact, for a later date.